src/Debugger.jl

```diff
@@ -27,7 +27,7 @@ end
 export @enter, @run
 
 # reexport some useful things from JuliaInterpreter
-using JuliaInterpreter: @bp, @breakpoint, breakpoint, break_on, break_off
+using JuliaInterpreter: @bp, @breakpoint, breakpoint, break_on, break_off, remove, disable
```

Comment:
Can we take these for another PR... I don't really like exporting these super-generic function names (ref https://github.com/JuliaDebug/JuliaInterpreter.jl/issues/281)

Reply:
ops, sorry I found those functions aren't defined, thus I suspected you just forgot to import them.
Do you want to still export them ?

Reply:
Just reverted for now :)

Reply:
Oh wait, this isn't exporting them. Then it's fine.

Reply:
you didn't import them, but exported

Reply:
Haha, okay!